Traceback (most recent call last):
  File "/home/vkeller/.local/lib/python3.9/site-packages/jupyter_cache/executors/utils.py", line 51, in single_nb_execution
    executenb(
  File "/home/vkeller/.local/lib/python3.9/site-packages/nbclient/client.py", line 1204, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
  File "/home/vkeller/.local/lib/python3.9/site-packages/nbclient/util.py", line 84, in wrapped
    return just_run(coro(*args, **kwargs))
  File "/home/vkeller/.local/lib/python3.9/site-packages/nbclient/util.py", line 62, in just_run
    return loop.run_until_complete(coro)
  File "/usr/lib/python3.9/asyncio/base_events.py", line 642, in run_until_complete
    return future.result()
  File "/home/vkeller/.local/lib/python3.9/site-packages/nbclient/client.py", line 663, in async_execute
    await self.async_execute_cell(
  File "/home/vkeller/.local/lib/python3.9/site-packages/nbclient/client.py", line 965, in async_execute_cell
    await self._check_raise_for_error(cell, cell_index, exec_reply)
  File "/home/vkeller/.local/lib/python3.9/site-packages/nbclient/client.py", line 862, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply_content)
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
import os
import pygame
import time
pygame.mixer.init()
pygame.mixer.music.load('bahn_cff_f.mp3')
pygame.mixer.music.play()
time.sleep(2.5)
train1 = ['S22', '8h47', 'Vufflens-la-Ville', 1]
annonce1 = 'say '+'Arriv√©e, sur voie '+str(train1[3])+', du train '+train1[0]+', √† destination de '+train1[2]+'. D√©part '+train1[1]
os.system(annonce1)
------------------

[0;31m---------------------------------------------------------------------------[0m
[0;31merror[0m                                     Traceback (most recent call last)
Input [0;32mIn [24][0m, in [0;36m<cell line: 5>[0;34m()[0m
[1;32m      3[0m [38;5;28;01mimport[39;00m [38;5;21;01mtime[39;00m
[1;32m      4[0m pygame[38;5;241m.[39mmixer[38;5;241m.[39minit()
[0;32m----> 5[0m [43mpygame[49m[38;5;241;43m.[39;49m[43mmixer[49m[38;5;241;43m.[39;49m[43mmusic[49m[38;5;241;43m.[39;49m[43mload[49m[43m([49m[38;5;124;43m'[39;49m[38;5;124;43mbahn_cff_f.mp3[39;49m[38;5;124;43m'[39;49m[43m)[49m
[1;32m      6[0m pygame[38;5;241m.[39mmixer[38;5;241m.[39mmusic[38;5;241m.[39mplay()
[1;32m      7[0m time[38;5;241m.[39msleep([38;5;241m2.5[39m)

[0;31merror[0m: No file 'bahn_cff_f.mp3' found in working directory '/home/vkeller/modulo-gybe/1M/prog/tableaux'.
error: No file 'bahn_cff_f.mp3' found in working directory '/home/vkeller/modulo-gybe/1M/prog/tableaux'.

